aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -1,49 +1,71 @@
 # Marketing App Resources
 # This template is manually deployed to create/update the Marketing App's AWS resources.
 
+<%
+site = Config::MARKETING_SITES[site_short_code.to_sym]
+BASE_DOMAIN_NAME = Config::STAGES[stage.to_sym][:baseDomain]
+
+LOAD_BALANCER_DOMAIN_NAME = "#{site[:subdomain]}-loadbalancer.#{BASE_DOMAIN_NAME}"
+CLOUDFRONT_DOMAIN_NAME = "#{site[:subdomain]}.#{BASE_DOMAIN_NAME}"
+%>
+
 AWSTemplateFormatVersion: 2010-09-09
 
 Description: Marketing App AWS resources
 
 Parameters:
-  BaseDomainName:
-    Type: String
-    Description: Base domain name (e.g. 'code.org' in 'marketing.code.org').
-  SubdomainName:
+  CloudFrontTLSCertificateArn:
     Type: String
-    Description: Subdomain name for marketing (e.g. 'marketing' in 'marketing.code.org').
-  Environment:
+    Description: The ARN of the TLS certificate for marketing app's cloudfront distribution. (e.g. the ARN of the ACM cert for 'test.marketing.code.org')
```

Comment:
I'd suggest instead using !ImportValue for this. In our CICD process, you name this stack `marketing-[site]`, and so you could name the certificate stack `marketing-[site]-certificate` and then you can use something like this when you need to reference it.

```
CertificateArn: !ImportValue 
  Fn::Sub: "${AWS::StackName}-certificate-CertificateArn"
```

Reply:
There would be a single certificate stack which creates all the certs within it. However, I could export the cert arn in a well known format and then do the ImportValue as suggested here.